OpenRA.Game/FileFormats/AudLoader.cs

```diff
@@ -153,7 +166,33 @@ public static byte[] LoadSound(Stream s)
 				dataSize -= 8 + chunk.CompressedSize;
 			}
 
-			return output;
+			rawData = output;
+			return true;
+		}
+
+		public bool TryParseSound(Stream stream, string fileName, out byte[] rawData, out int channels, out int sampleBits,
+			out int sampleRate)
+		{
+			channels = sampleBits = sampleRate = 0;
+
+			try
+			{
+				if (!LoadSound(stream, out rawData))
+					return false;
+			}
+			catch (Exception e)
+			{
+				Log.Write("debug", "Failed to parse AUD file {0}. Error message:".F(fileName));
```

Comment:
You're using TryParseSound for format detection as well as just loading.  This means you will spam the debug logs with "failed to load aud" errors for each valid wav if you define aud as the first loader in the list.